src/app/Sidebar/SidebarUsers.js

```diff
@@ -54,8 +67,12 @@ export default class SidebarMessages extends Component {
       );
     });
 
-    let contacts = this.props.contacts.slice(0, 20);
-    contacts = map(filter(contacts, (contact) => {
+    const search = this.state.search;
+    let users = filter(this.props.contacts, (user) => {
```

Comment:
filter is available in javascript why requiring additional lib? `this.props.contacts.filter(cb)`